doc/source/user/gdal_python_utilities.rst

```diff
@@ -0,0 +1,62 @@
+.. _gdal_python_utilities:
+
+================================================================================
+GDAL Python Utilities
+================================================================================
+
+The GDAL python utilities are included with GDAL. If you've installed
+GDAL you already have them. However you may want to use a newer or older
+version of the utilities without changing GDAL. This is where
+**gdal-utils** comes in.
+
+**gdal-utils**: is the GDAL Python Utilities *distribution*. This is
+what you install. It's home page is https://pypi.org/project/gdal-utils/
+. Install it with ``pip install gdal-utils`` and it will up or downgrade
+the utilities in place.
+
+**osgeo_utils**: is the python *package*. This is what you use in your
+code after installing, e.g. ``from osgeo_utils import ...``. If you're
+not writing code, ignore it.
+
+Commonly used utilities include:
+
+-  gdal_merge
+-  gdal_edit
+-  gdal_calc
+-  ogrmerge
+
+For the full list see
+`Programs <https://gdal.org/programs/index.html#programs>`__ on the GDAL
+website and note the ones that end in ``.py``.
```

Comment:
better use Sphinx anchors (see https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref).   The programs page has the ``programs`` anchor (see first line of https://raw.githubusercontent.com/OSGeo/gdal/master/doc/source/programs/index.rst)
```suggestion
:ref:`Programs <programs>` and note the ones that end in ``.py``.
```

Reply:
thanks!